packages/broker/src/ConfigWizard.ts

```diff
@@ -265,6 +265,16 @@ export const createStorageFile = async (config: any, answers: inquirer.Answers):
     return answers.selectDestinationPath
 }
 
+export const generateMiscFromConfig = (config: any) => {
```

Comment:
Could improve the naming of this function. Also the type of `config` parameter could be more specific.